### PR TITLE
Added chi weighted jump factor to the elliptic operator.

### PR DESCRIPTION
**Pull request by aslakp**
Possibility to weight the jump term by \chi in the elliptic operator.

### CODE REVIEW

File: inc/dg/Makefile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-device=gpu
+device=omp
```
Comment (mwiesenberger) [inline inc/dg/Makefile:1]: Should we change the default to omp? I don't care so much

**Review by mwiesenberger — CHANGES_REQUESTED** [review]
Hi Aslak,
looks good, could you add a documentation about what the weighted Jump term does in the elliptic equation in the class documentation, i.e. a Latex formula? I also see that you test the code in elliptic2d_b and elliptic_b, which is nice. What are the changes you observe regarding the convergence rate?

**Comment by mwiesenberger** [comment]
Also, does the addition of the weights preserve the symmetry of the operator?

File: inc/dg/Makefile (hunk 1 above)
Comment (aslakp) [reply inc/dg/Makefile:1]: Ohh yeah I forgot thi change, this was only to be able to run to code. I doesn't matter to me to be :)

Comment (aslakp) [reply inc/dg/Makefile:1]: Also, I'll make the change to documentation.

**Comment by aslakp** [comment]
Hi Matthias,
I changed the documentation to include the class documentation. I tried to be a concise as possible, but let me know if it is unclear.
In regards to the convergence, both cases seem to converge just fine. For the 3d case however, the weighted case need a few more iteration. For the 2D case, it is more or less the same. Note, this was not a rigorous testing.
In regards to the symmetry, I can't say for sure. Mathematically i believe it should be conserved, but I'm not sure how to check it numerically.

**Comment by mwiesenberger** [comment]
Regarding symmetry I think chi breaks it because (Chi J)^T = J^T Chi^T = J Chi, so maybe it should not be used in connection with CG?

**Pull request merged**